functions/awsmfa.fish

```diff
@@ -1,8 +1,8 @@
 function __awsmfa_test_expiry
     if test $AWS_SESSION_EXPIRY
 
-        set now (ruby -e "require 'time'; puts Time.now.to_i")
-        set expiry (ruby -e "require 'time'; puts Time.iso8601('$AWS_SESSION_EXPIRY').to_i")
+        set now (date +'%s')
+        set expiry (date -j -f "%FT%TZ" "$AWS_SESSION_EXPIRY" +%s)
```

Comment:
`-j` is only available on macOS (and I'm assuming BSDs), I'm pretty sure that's why I ended up generating that in ruby since GNU date doesn't have it. What's the error in you get with ruby?

Reply:
Ah of course.  

The error I was getting was like this 

```Traceback (most recent call last):
        1: from /usr/local/bin/bundle:23:in `<main>'
/usr/local/bin/bundle:23:in `load': cannot load such file -- /usr/local/lib/ruby/gems/2.6.0/gems/bundler-1.17.3/exe/bundle (LoadError)
Traceback (most recent call last):
        1: from -e:1:in `<main>'
/usr/local/Cellar/ruby/2.6.1/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:34:in `require': wrong number of arguments (given 0, expected 1) (ArgumentError)
fish: Unknown command puts
- (line 1):
begin; command ruby -e require 'time'; puts Time.now.to_i
                                       ^
from sourcing file -
        called on line 61 of file /usr/local/Cellar/fish/3.0.2/share/fish/functions/eval.fish

in function 'eval'
        called on line 6 of file ~/.local/share/omf/pkg/bundler/functions/__execute_as_bundler.fish

in function '__execute_as_bundler'
        called on line 2 of file -
        with parameter list 'ruby -e require 'time'; puts Time.now.to_i'

in function 'ruby'
        called on line 2 of file ~/.local/share/omf/pkg/awsmfa/functions/awsmfa.fish
        with parameter list '-e require 'time'; puts Time.now.to_i'

in command substitution
        called on line 5 of file ~/.local/share/omf/pkg/awsmfa/functions/awsmfa.fish

in function '__awsmfa_test_expiry'
        called on line 69 of file ~/.local/share/omf/pkg/awsmfa/functions/awsmfa.fish

in function 'awsmfa'
        called on standard input
        with parameter list 'XXXX_PROFILE'
```